test/suites/ACID.m

```diff
@@ -2379,6 +2380,38 @@
     set(h_t,'BackgroundColor','g');
 end
 % =========================================================================
+function [stat] = overlappingPlots()
+    stat.description = 'Overlapping plots with zoomed data and varying background.';
+    stat.issues = 6;
+
+    % create pseudo random data and convert it from matrix to vector
+    l = 256;
+    l_zoom = 64;
+    wave = sin(linspace(1,10*2*pi,l));
+
+    % plot data
+    h = figure(); clf;
+    ax1 = axes('Parent', h);
+    plot(ax1, wave);
+
+    % overlapping plots with zoomed data
+    ax3 = axes('Parent', h, 'Position', [0.2, 0.6, 0.3, 0.4]);
+    ax4 = axes('Parent', h, 'Position', [0.7, 0.2, 0.2, 0.4]);
+    ax2 = axes('Parent', h, 'Position', [0.25, 0.3, 0.3, 0.4]);
+
```

Comment:
@PeterPablo My bad, I overlooked the fact we should not create figures inside the testing sub-fuctions since `testM2t` already does that. Now, at the end of the full ACID, the last test is created and the figure pops into visibility. 

The test case should be:
```matlab
% plot data
ax1 = axes()
plot(ax1, wave);

% overlapping plots with zoomed data
ax3 = axes('Position', [0.2, 0.6, 0.3, 0.4]);
ax4 = axes('Position', [0.7, 0.2, 0.2, 0.4]);
ax2 = axes('Position', [0.25, 0.3, 0.3, 0.4]);
```